plugins/h3/h3.cpp

```diff
@@ -39,7 +39,7 @@ ECL_H3_API bool getECLPluginDefinition(ECLPluginDefinitionBlock *pb)
     pb->moduleName = "lib_h3";
     pb->ECL = NULL;
     pb->flags = PLUGIN_IMPLICIT_MODULE;
-    pb->description = "ECL plugin library for uber h3\n";
```

Comment:
Note - these don't actually appear in the logs - I initially output this too but decided it was too verbose. But I decided to make them consistent wrt \n anyway.